babel.cfg

```diff
@@ -0,0 +1,2 @@
+[python: app/**.py]
+[jinja2: **/templates/**.html]
```

Comment:
Do we need both `babel.cfg` and `config/babel.cfg` what is the difference? Can one be deleted?

Reply:
sorry they're the same file and the config one should stay, I'm removing the root babel file 👍🏾

Reply:
@gidsg I didn't notice that, it was for tidiness. Shall we move to root?

Reply:
Moving it to the root for now as discussed, as so to be consistent with documentation: https://github.com/communitiesuk/funding-service-design-utils#translations